guides/nestor/ISPsetup.md

```diff
@@ -0,0 +1,308 @@
+---
+Layout: post
+Title: ISP setup ubuntu
+Author: Nestor N Torres
+---
+
+### Contact
+- Slack: @nanjuan on wcscusf.slack.com
+- [LinkedIn](https://www.linkedin.com/in/nestor-n-torres-737172a5/)
+- Email: nestor@nntorres.com
+
+### Table
+1. [Prerequisites](#id-link-to-section)
+2. [Summary](#Summary)
+3. [First logical group of Page](#First)
+4. [Second logical group of Page](#Second)
+5. [Third Logical group of pare](#Third)
+
+## Prerequisites <a id="id-link-to-section"></a>
+1. If you have not setup your network adapter look here https://goo.gl/vs4cHC.
+2. Just copy the Prerequisites of peoples guides before yours here.
+3. Have your [virtual environment](link-to-guide) configured
+4. Have the [ISP](link-to-guide) gateway running.
+5. Have [pfSense](link-to-guide) running.
+
+## Summary <a id="Summary"></a>
+This tutorial will show you how to setup the ISP server you will need for your lab.
+
+<br>
+
+| Network adapters | Description                     |
+|------------------|---------------------------------|
+| Adapter 1        | Leave it to Nat                 |
+| Adapter 2        | Change to Host-only (vboxnet2)  |
+
```

Comment:
Images would be awesome here. Where does adapter 1 located at and where is adapter 2 located at. Where is the ISP located in the topology.

Reply:
how can you make photos be small not full size??

Reply:
```
<figure>
    <img src="logo.gif" width="500px" height="300px" alt="USF logo">
    <figcaption>title displayed</figcaption>
</figure>
```